explore/src/main/scala/explore/targeteditor/ElevationPlotNight.scala

```diff
@@ -210,7 +210,10 @@ object ElevationPlotNight {
             case TimeDisplay.UT       => timezoneInstantFormat(instant, ZoneOffset.UTC)
             case TimeDisplay.Sidereal =>
               val skycalc = ImprovedSkyCalc(props.site.place)
-              val sid     = skycalc.getSiderealTime(instant)
+              val sid     = {
+                val sid = skycalc.getSiderealTime(instant)
+                if (sid < 0) sid + 24 else sid
```

Comment:
Is sid constrained to +/- 24?

Reply:
Yes, it is not very clearly stated on the type :D